src/sections/Why.vue

```diff
@@ -1,16 +1,228 @@
+              things we learned. We'll host live workshops
+              all weekend to give you the inspiration you
+              need to get your project off the ground.
+            </Typography>
+            <img class="why__gears" src="../assets/Why/learn/gears.svg">
+        </div>
+        <div class="why__card">
+          <Placeholder class='why__placeholder' width='385px' height='385px'/>
+            <Typography class='why__card-heading' type='heading3' color='white' as='h3'>
+              Want to be rewarded
+              for your hard work?
+            </Typography>
+            <Typography class='why__card-body' type='xsmall' color='white' as='p'>
+              With $38,000 worth of prizes, there’s
+              something for everyone.
+            </Typography>
+            <img class="why__prize" src="../assets/Why/reward/prize.svg">
+        </div>
+        <div class="why__card">
+          <Placeholder class='why__placeholder' width='385px' height='385px'/>
+            <Typography class='why__card-heading' type='heading3' color='white' as='h3'>
+              Need projects on your
+              portfolio? 
+            </Typography>
+            <Typography class='why__card-body' type='xsmall' color='white' as='p'>
+              Complete a project worth showcasing within
+              48 hours from scratch and land your next job.
+              Check out what our hackers created last year!
+            </Typography>
+        </div>
+        <div class="why__card">
+          <Placeholder class='why__placeholder' width='385px' height='385px'/>
+            <Typography class='why__card-heading' type='heading3' color='white' as='h3'>
+              Feeling stuck at home?
+            </Typography>
+            <Typography class='why__card-body' type='xsmall' color='white' as='p'>
+              Physical distancing is serious, but meeting
+              new people doesn't have to stop. Get geared
+              up in your pajamas and make tons of virtual
+              memories. 
+            </Typography>
+            <img class="why__memories" src="../assets/Why/home/memories.svg">
+        </div>
+      </div>
+      <div class='why__button'>
+        <Button class='why__left' @click='changeSlide(-1)'>
+          <svg width="30px" height="105px" viewBox="0 0 28 74" stroke="teal" stroke-width="8" stroke-linecap="round" stroke-linejoin="round" xmlns="http://www.w3.org/2000/svg">
+            <line x1="4" y1="37" x2="24" y2="4"/>
+            <line x1="4" y1="37" x2="24" y2="70"/>
+          </svg>
+        </Button>
+        <Button class='why__right' @click='changeSlide(1)'>
+          <svg width="30px" height="105px" viewBox="0 0 28 74" stroke="teal" stroke-width="8" stroke-linecap="round" stroke-linejoin="round" xmlns="http://www.w3.org/2000/svg">
+              <line x1="24" y1="37" x2="4" y2="4"/>
+              <line x1="24" y1="37" x2="4" y2="70"/>
+          </svg>
+        </Button>
```

Comment:
You can import the svgs as components because of the webpack plugin we have added. Here is an example,
https://github.com/hack-the-6ix/hackthe6ix-landing-2021/blob/develop/src/sections/About.vue#L142